src/utils/api.js

```diff
@@ -54,3 +53,16 @@ export async function getLatestTopics(req) {
     return []
   }
 }
+
+export async function getCommentsCount(commentsUrl) {
+  try {
+    const res = await fetch(`/api/comments?url=${commentsUrl}`)
+
+    if (res.status === 200) {
+      const { count } = await res.json()
+      return count
+    }
+  } catch (e) {
+    console.error(e)
```

Comment:
Out of scope: maybe we should reject the promise when there's a non-200 response?

Reply:
It's mostly a question of there to handle rejections. We can do it either here or in components themselves. For now I decided that it will probably be easier to do there them increase components size.

P. S. I like your idea to transforms API calls to custom hooks, it will solve this problem and they can also have error state in them that components can use. I will try to convert them to custom hooks now.

Reply:
I see what you mean.

But maybe a non-200 response should have the same treatment as an error. This decreases the amount of outcomes the function may have. Right now, it's [error, undefined, response]. If we treat non-200 as an error it's only [error, response].